main/WebServer.cpp

```diff
@@ -6784,7 +6784,7 @@ namespace http {
 						root["status"] = "OK";
 						root["title"] = "LearnSW";
 						root["ID"] = m_sql.m_LastSwitchID;
-						root["idx"] = m_sql.m_LastSwitchRowID;
+						root["idx"] = Json::Value::UInt64(m_sql.m_LastSwitchRowID);
```

Comment:
I don't have any warnings here...
It's already a uint64_t 
Is it the JsonCPP version maybe ? As mine Json::Value::UInt64 points to a unsigned __int64 -> uint64_t

Reply:
I think so. Either that or GCC8 is too stupid to figure it out.